ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/MapContainerViewModel.kt

```diff
@@ -183,35 +171,17 @@ internal constructor(
     }
   }
 
-  fun onMapDrag() {
-    if (isLocationLockEnabled()) {
-      Timber.d("User dragged map. Disabling location lock")
-      locationController.unlock()
-    }
-  }
-
   fun onMarkerClick(mapLocationOfInterest: MapLocationOfInterest) {
-    val locationOfInterest = mapLocationOfInterest.locationOfInterest
-    if (locationOfInterest != null) {
-      val geometry = locationOfInterest.geometry
-      if (geometry is Point) {
-        mapController.panAndZoomCamera(geometry)
-      }
+    val geometry = mapLocationOfInterest.locationOfInterest.geometry
```

Comment:
Change LG. Was any other logic changes besides this? Was moved/refactored code moved without changes? It's hard to tell from the diff for code that was moved/refactored.

Reply:
This was flagged as "always true" by the IDE since LOI can't be null. This was the only logical change apart from the refactor